examples/simple/pi.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Dredd Project Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+
+using namespace std;
+
+static double calculatePi() {
+    double sum = 0.0;
+    int numerator = -1;
+
+    for (int d = 1; d < 1000000; d += 2) {
+        numerator *= -1;
+        sum += numerator / double(d);
+    }
+
+    return sum * 4;
+}
+
+int main() {
+    cout << calculatePi() << "\n";
```

Comment:
I suggest removing the `using std`, changing `cout` to `std::cout`, and changing "\n" to `std::endl`